Screens/LineupForm.js

```diff
@@ -0,0 +1,189 @@
+      precipitation: propState.precipitation,
+      course:        propState.course,
+      racers:        propState.racers,
+      position:      propState.position,
+      images: [],
+      lane1: '',
+      lane2: '',
+      lane3: '',
+      lane4: '',
+      lane5: '',
+      lane6: '',
+    }
+  }
+
+  setImages = (data) => {
+    this.setState({
+      images:
+        [...this.state.images,
+          {
+            source: data.mediaUri
+          }
+        ]
+      })
+    console.log(this.state.images)
+  }
+
+  renderImages() {
+    if (this.state.images.length < 1) {
+      return (
+        <Text style={styles.imageText}>
+          Use the camera to capture images
+        </Text>
+      )
+    } 
+    else {
+      return (
+        <View style={styles.imageGallery}>
+          <Image
+            style={styles.image}
+            source={{uri: this.state.images[0].source}}
+          />
+        </View>
+      )
+    }
+  }
+
+  render() {
+    const { navigate } = this.props.navigation
+    return (
+      <View style={styles.container}>
+        <View style={styles.imageView}>
+          {this.renderImages()}
+        </View>
+        <Button
+          onPress={() =>
+            navigate('ImageCapture', { setImages: this.setImages })
+          }
+          style={styles.button}
+          title='Camera'
+        />
+        <View style={styles.row}>
+          <Text style={styles.text}>
+            Lane 1:
+          </Text>
+          <Picker style={styles.input}
+          />
+        </View>
+        <View style={styles.row}>
+          <Text style={styles.text}>
+            Lane 2:
+          </Text>
+          <Picker style={styles.input}
+          />
+        </View>
+        <View style={styles.row}>
+          <Text style={styles.text}>
+            Lane 3:
+          </Text>
+          <Picker style={styles.input}
+          />
+        </View>
+        <View style={styles.row}>
+          <Text style={styles.text}>
+            Lane 4:
+          </Text>
+          <Picker style={styles.input}
+          />
+        </View>
+        <View style={styles.row}>
+          <Text style={styles.text}>
+            Lane 5:
+          </Text>
+          <Picker style={styles.input}
+          />
+        </View>
+        <View style={styles.row}>
+          <Text style={styles.text}>
+            Lane 6:
+          </Text>
+          <Picker style={styles.input}
+          />
+        </View>
+        <Button
+          onPress={() => {
+              console.log(this.state)
+              this.state.images = []
+            }
+          }
+          style={styles.button}
+          title='Submit'
+        />
+      </View>
+    )
+  }
+}
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    flexDirection: 'column',
+  },
+  row: {
+    flex: 1,
+    flexDirection: 'row',
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
+  image: {
+    width: width,
+    height: height / 3,
+  },
```

Comment:
Seems strange to set absolute width and height of an image based on screen dimensions, which can be quite variable. I don't know any off the top of my head, but I'm sure there's a better way to size an image.

Reply:
If you are going to use the dimensions of the phone to set the photo size - you should make sure you pick the correct prop for `resizeMode` (i.e., 'cover' in this case, which may be the default setting)

---
In reply to: [157106272](https://github.com/rozele/React-Native-Ski-Team/pull/2#discussion_r157106272) [](ancestors = 157106272)